sql/database.sql

```diff
@@ -2,7 +2,11 @@
 -- Database: `openemr`
 --
 
------------------------------------------------------------
+--
+-- Disable FOREIGN_KEY_CHECKS
+--
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
probably a dumb question,
Why is this used here, but not in the upgrade script?

Reply:
Because need for drop table. Create Table is okay.